Src/Support/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -40,6 +41,12 @@ public class GoogleAuthorizationCodeFlow : AuthorizationCodeFlow
         /// <summary>Gets or sets the include granted scopes indicator.</summary>
         public bool? IncludeGrantedScopes { get { return includeGrantedScopes; } }
 
+        private readonly IEnumerable<KeyValuePair<string, string>> userDefinedQueryParams;

```

Comment:
Yup, good catch, thanks - lost in the move to the new PR. I'll look in to the unit test next time I can sit down.